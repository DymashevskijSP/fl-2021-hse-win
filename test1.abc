Sigma:"0,1" 
Q:"q_0,q_1"
q0:"q_0"
Terminals: "q_1"
delta:'(q_0, 1)->q_1, (q_0, 0)->q_0, (q_1, 1)->q_1, (q_1, 0)->q_0'
