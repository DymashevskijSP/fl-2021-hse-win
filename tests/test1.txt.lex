LexToken(RULENAME,'A',1,0)
LexToken(EQ,'=',1,1)
LexToken(PLAINTEXT,'C',1,2)
LexToken(ALT,'|',1,5)
LexToken(PLAINTEXT,'B',1,6)
LexToken(COMMA,',',1,9)
LexToken(PLAINTEXT,'C',1,10)
LexToken(ALT,'|',1,13)
LexToken(PLAINTEXT,'A',1,14)
LexToken(COMMA,',',1,17)
LexToken(LPAREN,'(',1,18)
LexToken(PLAINTEXT,'B',1,19)
LexToken(RPAREN,')',1,22)
LexToken(RULENAME,'B',2,24)
LexToken(EQ,'=',2,25)
LexToken(PLAINTEXT,'D',2,26)
LexToken(ALT,'|',2,29)
LexToken(PLAINTEXT,'C',2,30)
LexToken(COMMA,',',2,33)
LexToken(LBRACKET,'[',2,34)
LexToken(PLAINTEXT,'A',2,35)
LexToken(RBRACKET,']',2,38)
LexToken(COMMA,',',2,39)
LexToken(LBRACE,'{',2,40)
LexToken(PLAINTEXT,'B',2,41)
LexToken(RBRACE,'}',2,44)
