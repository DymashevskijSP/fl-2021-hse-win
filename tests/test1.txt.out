Node:    A =  C |  B ,  C |  A ,  ( B )  B =  D |  C ,   [ A ] ,  { B }: Children   A =  C |  B ,  C |  A ,  ( B ) |  B =  D |  C ,   [ A ] ,  { B } |
Node:   A =  C |  B ,  C |  A ,  ( B ): Children  A =  C |  B ,  C |  A ,  ( B ) |
Node:  B =  D |  C ,   [ A ] ,  { B }: Children B | = |  D |  C ,   [ A ] ,  { B } |
Node:  A =  C |  B ,  C |  A ,  ( B ): Children A | = |  C |  B ,  C |  A ,  ( B ) |
Node: B: Children EMPTY
Node: =: Children EMPTY
Node:  D |  C ,   [ A ] ,  { B }: Children D | | |  C ,   [ A ] ,  { B } |
Node: A: Children EMPTY
Node: =: Children EMPTY
Node:  C |  B ,  C |  A ,  ( B ): Children C | | |  B ,  C |  A ,  ( B ) |
Node: D: Children EMPTY
Node: |: Children EMPTY
Node:  C ,   [ A ] ,  { B }: Children C | , |   [ A ] ,  { B } |
Node: C: Children EMPTY
Node: |: Children EMPTY
Node:  B ,  C |  A ,  ( B ): Children B | , |  C |  A ,  ( B ) |
Node: C: Children EMPTY
Node: ,: Children EMPTY
Node:   [ A ] ,  { B }: Children  [ A ] | , |  { B } |
Node: B: Children EMPTY
Node: ,: Children EMPTY
Node:  C |  A ,  ( B ): Children C | | |  A ,  ( B ) |
Node:  [ A ]: Children [ | A | ] |
Node: ,: Children EMPTY
Node:  { B }: Children { | B | } |
Node: C: Children EMPTY
Node: |: Children EMPTY
Node:  A ,  ( B ): Children A | , |  ( B ) |
Node: [: Children EMPTY
Node: A: Children EMPTY
Node: ]: Children EMPTY
Node: {: Children EMPTY
Node: B: Children EMPTY
Node: }: Children EMPTY
Node: A: Children EMPTY
Node: ,: Children EMPTY
Node:  ( B ): Children ( | B | ) |
Node: (: Children EMPTY
Node: B: Children EMPTY
Node: ): Children EMPTY